x/group/spec/group.md

```diff
@@ -66,9 +70,10 @@ to authorize messages send back to the router.
 
 ```go
 type GroupKeeper interface {
-  IterateGroupsByMember(member sdk.Address, fn func (group sdk.AccAddress) (stop bool))
-  IterateGroupsByAdmin(member sdk.Address, fn func (group sdk.AccAddress) (stop bool))
-  GetGroupDescription(group sdk.AccAddress) string
-  GetTotalPower(group sdk.AccAddress) sdk.Int
+  IterateGroupMembers(group GroupID, fn func (member sdk.AccAddress, weight sdk.Int, description string) (stop bool))
```

Comment:
Okay, this is nice and specifies how to load the members. Much clearer now, thanks.